deploy/helm/charts/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: lvm-localpv
 description: CSI Driver for dynamic provisioning of LVM Persistent Local Volumes.
-version: 1.6.1
+version: 1.6.2
 appVersion: 1.6.1
```

Comment:
Shouldn't the app version also change?

Reply:
No, the code sees no change, the appVersion (and container image version) are the same. Changes are going into the helm chart exclusively.